android/src/main/java/com/rudderstack/react/android/RNLifeCycleEventListener.java

```diff
@@ -0,0 +1,68 @@
+package com.rudderstack.react.android;
+
+import android.app.Activity;
+
+import com.facebook.react.bridge.LifecycleEventListener;
+import com.rudderstack.android.sdk.core.RudderProperty;
+import com.rudderstack.android.sdk.core.ScreenPropertyBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RNLifeCycleEventListener implements LifecycleEventListener {
+
+    private static int noOfActivities;
+    static List<Runnable> runnableTasks = new ArrayList<>();
+
+    @Override
+    public void onHostResume() {
+        Runnable runnableTask = new Runnable() {
+            @Override
+            public void run() {
+                if (RNRudderSdkModule.trackLifeCycleEvents) {
+                    noOfActivities += 1;
+                    if (noOfActivities == 1) {
+                        // no previous activity present. Application Opened
+                        RNRudderSdkModule.rudderClient.track("Application Opened");
```

Comment:
Are we not sending the properties like [iOS](https://github.com/rudderlabs/rudder-sdk-ios/blob/d875bb1598b23f3ba7c1c8faa5ac8b5900329cc1/Sources/Classes/RSEventRepository.m#L612)?

Reply:
Yeah, we are not sending properties previously as well. But thank you for pointing it out, will add the support for sending properties as well

Reply:
Thanks.

Reply:
Now added the support for sending properties as well